packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -155,10 +161,30 @@ const ProductDetail = ({category, product, isLoading}) => {
                         <ProductView
                             product={product}
                             category={primaryCategory?.parentCategoryTree || []}
-                            addToCart={(variant, quantity) => handleAddToCart(variant, quantity)}
+                            addToCart={() => {
+                                const productSelectionValues = Object.values(productSetSelection)
+
+                                return handleAddToCart(productSelectionValues)
+                            }}
                             addToWishlist={(_, quantity) => handleAddToWishlist(quantity)}
                             isProductLoading={isLoading}
                             isCustomerProductListLoading={!wishlist.isInitialized}
+                            validate={() => {
```

Comment:
Revisiting what we talked in our meeting earlier: what to do when a child product is not in stock? Currently shoppers would not able to add all the items in the set because `validate` would return false.

Reply:
As per our original meeting Nathan we are going to wait for some feedback and iterate from there.